java-client/src/test/java/org/opensearch/client/opensearch/integTest/AbstractClusterClientIT.java

```diff
@@ -95,11 +103,19 @@ public void testClusterUpdateSettingNonExistent() throws IOException {
         } catch (OpenSearchException e) {
             assertNotNull(e);
             assertEquals(e.response().status(), 400);
-            assertEquals(
+            if (Version.fromString(version).onOrAfter(Version.fromString("3.0.0"))) {
+                assertEquals(
+                    e.getMessage(),
+                    "Request failed: [settings_exception] " +
```

Comment:
Do we need the exact match here? May be simpler check would be to ensure `"transient setting [no_idea_what_you_are_talking_about], not recognized"` is present?